src/components/settingsBadge/SettingsBadge.tsx

```diff
@@ -0,0 +1,38 @@
+import { useSelector } from "react-redux";
+import IconComponent from "../iconComponent/IconComponent";
+import s from "./settingsBadge.module.scss";
+import { selectBasicSettings } from "../../redux/selectors";
+import { useState } from "react";
+import ModalWrapper from "../modalWrapper/ModalWrapper";
+
+interface BadgeProps {
+  // onClick: () => void;
+}
+
+function SettingsBadge({}: BadgeProps) {
+  const basicSettings = useSelector(selectBasicSettings);
+  const [isMicAndCamOn, setIsMicAndCamOn] = useState({
+    isMicOn: basicSettings.microphoneOn,
+    isCamOn: basicSettings.camera,
+  });
+  const [showModal, setShowModal] = useState(false);
+
+  const onClick = () => {
```

Comment:
We follow the style code, if we set types for functions, then for all of them

Reply:
fix